### PR TITLE
Fix #14:Clean up update_game_state

### LINKED ISSUE #14: check_game_state subroutine is not called from anywhere
Hi @flxbe !

check_game_state subroutine in game.asm is not called from anywhere.

A couple of thoughts:

1. To Delete to check_game_state subroutine in game.asm.
2. To Call check_game_state subroutine from space-invaders.asm.

Which is better?

Best regards,

### PR DESCRIPTION
**Pull request by cyblue9**
It corresponds to #14.

Please review!

### CODE REVIEW

File: space-invaders.asm [hunk 1]
```diff
@@ -20,9 +20,10 @@
 %define ICON_EXPLOSION_BULLET '#'
 %define ICON_WALL '#'
 
-; GAME WINNERS
-%define PLAYER_WIN 0
-%define INVADERS_WIN 1
+; GAME STATES
+%define STILL_PLAYING 0
+%define PLAYER_WIN 1
+%define INVADERS_WIN 2
```
Comment (flxbe) [inline space-invaders.asm]: As before, I would encode "type" information as a prefix within the names.

```asm
%define GAME_STATE_PLAYING 0
%define GAME_STATE_PLAYER_WINS 1
%define GAME_STATE_INVADERS_WIN 2
```

**Review by flxbe — CHANGES_REQUESTED** [review]
Only a minor remark.

File: space-invaders.asm (hunk 1 above)
Comment (flxbe) [inline space-invaders.asm]: These constants should also include "type" information in the name.

```asm
%define GAME_STATE_PLAYING 0
%define GAME_STATE_PLAYER_WINS 1
%define GAME_STATE_INAVERS_WIN 2
```

**Review by flxbe — CHANGES_REQUESTED** [review]
Only a minor remark.

File: space-invaders.asm (hunk 1 above)
Comment (cyblue9) [reply space-invaders.asm]: Sorry, I forgot it...

Comment (blue-9) [reply space-invaders.asm]: Sorry, I forgot it ...

Comment (cyblue9) [reply space-invaders.asm]: Does it have the same meaning as above(https://github.com/flxbe/asm-space-invaders/pull/15#discussion_r226906744)?

**Comment by cyblue9** [comment]
I fixed. 

Please, Re-review!

**Review by flxbe — APPROVED** [review]

**Pull request merged**